<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 40pt &quot;Avenir&quot;;
background-color: rgb(33, 59, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string>Brandon</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string>Phillennia</string>
    </property>
   </widget>
   <widget class="QLabel" name="brandonScore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="phillyScore">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="secondScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="thirdScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourthScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fifthScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixthScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="seventhScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="eighthScore">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>500</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Avenir&quot;;
color: rgb(209, 207, 212);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="brandonAddPoints">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 191, 195);
font: 27pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="phillyAddPoints">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 191, 195);
font: 27pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="brandonButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(221, 218, 224);
background-color: rgb(74, 208, 206);

font: 18pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="phillyButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(221, 218, 224);
background-color: rgb(74, 208, 206);
font: 18pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(221, 218, 224);
background-color: rgb(255, 131, 129);

font: 18pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
